lib/partial/transmuxer.js

```diff
@@ -0,0 +1,375 @@
+var Stream = require('../utils/stream.js');
+var m2ts = require('../m2ts/m2ts.js');
+var codecs = require('../codecs/index.js');
+var AudioSegmentStream = require('./audio-segment-stream.js');
+var VideoSegmentStream = require('./video-segment-stream.js');
+var trackInfo = require('../mp4/track-decode-info.js');
+var isLikelyAacData = require('../aac/utils').isLikelyAacData;
+var AdtsStream = require('../codecs/adts');
+var AacStream = require('../aac/index');
+var clock = require('../utils/clock');
+
+var createPipeline = function(object) {
+  object.prototype = new Stream();
+  object.prototype.init.call(object);
+
+  return object;
+};
+
+var tsPipeline = function(options) {
+  var
+    pipeline = {
+      type: 'ts',
+      tracks: {
+        audio: null,
+        video: null
+      },
+      packet: new m2ts.TransportPacketStream(),
+      parse: new m2ts.TransportParseStream(),
+      elementary: new m2ts.ElementaryStream(),
+      videoRollover: new m2ts.TimestampRolloverStream('video'),
+      audioRollover: new m2ts.TimestampRolloverStream('audio'),
+      adts: new codecs.Adts(),
+      h264: new codecs.h264.H264Stream(),
+      captionStream: new m2ts.CaptionStream(),
+      metadataStream: new m2ts.MetadataStream(),
+      timedMetadataTimestampRolloverStream:
```

Comment:
Bit of a mouthful compared to the rest of the stream properties. Maybe `timedMetadataRollover` to match the audio and video versions